app/controllers/users_controller.rb

```diff
@@ -1,10 +1,35 @@
 class UsersController < ApplicationController
-  skip_before_action :authenticate_user!, only: %i[index show]
+  before_action :authenticate_user!, except: [:index, :show]
+  before_action :ensure_chef, only: [:edit, :update]
+
   def index
     @chefs = User.all
   end
 
   def show
     @chef = User.find(params[:id])
   end
+
+  def edit
+    @chef = current_user
```

Comment:
This line is repeated twice. It can be put into the ensure_chef to keep the code dry

Reply:
sorry, im confused with this part